wallets/provider-tron-link/src/index.ts

```diff
@@ -23,27 +30,63 @@ export const config = {
   defaultNetwork: Networks.TRON,
 };
 
-export const getInstance = tronLink_instance;
+export const getInstance = tronLinkInstances;
+
+export const connect: Connect = async ({ instance, meta }) => {
+  const ethInstance = chooseInstance(instance, meta, Networks.ETHEREUM);
+  const tronInstance = chooseInstance(instance, meta, Networks.TRON);
 
-export const connect: Connect = async ({ instance }) => {
-  let r = undefined;
-  if (!!instance && !instance.ready) {
-    r = await instance.request({ method: 'tron_requestAccounts' });
-    if (!r) {
+  const results: ProviderConnectResult[] = [];
+
+  if (ethInstance) {
```

Comment:
It tries to connect to Ethereum and Tron (instance) at the same time but the wallet seems doesn't support that and one network should be connected at the same time.